test/Microsoft.Azure.ServiceBus.UnitTests/BrokeredMessageTests.cs

```diff
@@ -19,7 +19,11 @@ public class BrokeredMessageTests
                 ReceiveMode.ReceiveAndDelete);
 
             await TestUtility.SendMessagesAsync(queueClient.InnerSender, 1);
-            var message = await queueClient.ReceiveAsync();
+            var message = await queueClient.PeekAsync();
```

Comment:
Agree, since the tests were already taking time, I tried to reduce the time for another sent message and another receive etc by including that in there. :-)... But agree for clarity this is better, I've seperated that out and published a new pull request.